public/templates/public/blocks/related_exhibit.html

```diff
@@ -11,16 +11,20 @@
   {% if self.exhibit %}
     <a href="{% pageurl self.exhibit %}" class="event-header">{{self.exhibit}}</a>
     {% if self.exhibit.exhibit_open_date or self.exhibit.exhibit_close_date %}
-      <br/>{{self.exhibit.exhibit_open_date}}
-      {% if self.exhibit.exhibit_open_date and self.exhibit.exhibit_close_date %}
-        —
+      {% if self.exhibit.is_online_exhibit %}
+         <br/>Online from {{self.exhibit.exhibit_open_date}}
+      {% else %}
+        <br/>{{self.exhibit.exhibit_open_date}}
```

Comment:
For this one, I was thinking of something more like this (which says the same thing as what you have):

```
      <br/>
      {% if self.exhibit.is_online_exhibit %}
         Online from {{self.exhibit.exhibit_open_date}}
      {% else %}
        {{self.exhibit.exhibit_open_date}}
```

However, now that I look at this closer, I think you might have intended to do something like this:

```
      {% if self.exhibit.is_online_exhibit and self.exhibit.exhibit_open_date%}
        <br/>Online from {{self.exhibit.exhibit_open_date}}
      {% elif self.exhibit.exhibit_open_date %}
        <br/>{{self.exhibit.exhibit_open_date}}
```

In the second example it's good to leave the `<br/>`s inside the `if elif` clause, because they only display if the data is present. In the first example, however, there will _always_ be a `<br/`>` in the page, even if there is _not_ an `exhibit_open_date`. When I look at this closer, I feel like that wasn't intended because we could actually have a case where it would try to put the open date in the template even if it wasn't present. I think maybe the second one is what we want. Does that make sense?